### PR TITLE
[WIP] Swarm guide

### PR DESCRIPTION
**Pull request by rgbkrk**
I apparently had this branch sitting around on running nature directly with a Docker setup. This one runs directly on individual swarm hosts, pulling keys from particular containers. It's not meant to be secure, but highlights a way of stitching this together.

### CODE REVIEW

**Comment by rgbkrk** [comment]
This is missing the far more insecure option which is to set `container_ip` to `0.0.0.0`.

**Comment by captainsafia** [comment]
Thanks for making that `docs` folder. 

Who is the intended audience for this file?

File: docs/nature-on-swarm.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+           -e DOCKER_HOST="tcp://127.0.0.1:42376" \
+           -e DOCKER_TLS_VERIFY=1 \
+           -e DOCKER_CERT_PATH=/etc/docker \
+           jupyter/tmpnb python orchestrate.py --image='jupyter/nature-demo' \
+           --redirect_uri='/notebooks/Nature.ipynb' \
+           --command="ipython3 notebook --NotebookApp.base_url={base_path} --ip=0.0.0.0 --port {port}" \
+           --pool_size=$POOL_SIZE \
+           --mem_limit='512m' \
+           --cpu_shares=$CPU_SHARES
+```
```
Comment (captainsafia) [inline docs/nature-on-swarm.md]: Might want to move this to the end of the file.

**Comment by rgbkrk** [comment]
> Who is the intended audience for this file?

:expressionless: right now, me, @odewahn, and @zischwartz.

**Comment by rgbkrk** [comment]
I'll go ahead and close this one, only leaving my branch on my fork for reference.

**Pull request closed without merging**